src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -356,85 +356,25 @@ Once a MLaaS instance is up and running, it could be used for:
 Using MLaaS, you can easily deploy and use any machine learning model as a REST Web Service on a physical or a virtual compute host on which there is an available Proactive Node. Going through the Proactive Scheduler,
 you can also trigger the deployment of a specific VM using the Resource Manager elastic policies, and eventually, deploy a Model-Service on that specific node.
 
-The following deployment steps are similar wherever you need to deploy and use your model:
+In the following subsections, we will illustrate the life cycle of MLaaS instance starting from the generic service instance,
```

Comment:
of MLaaS instance => of an (or a) MLaaS instance

Reply:
starting from the generic service instance => maybe you should put the same form as the next sentences. A suggestion:
from starting the generic service instance, deploying a specific model, pausing it, to deleting the instance. We will also talk about how these ...